CHANGELOG.md

```diff
@@ -27,6 +45,7 @@ This is a changelog for Piwik platform developers. All changes for our HTTP API'
 * The new class `Piwik\Category\Subcategory` let you change the name and order of menu items
 * New HTTP API method `API.getWidgetMetadata` to get a list of available widgets
 * New HTTP API method `API.getReportPagesMetadata` to get a list of all available pages that exist including the widgets they include
+* New event `ViewDataTable.filterViewDataTable` let's you filter available visualizations
```

Comment:
I removed the old `ViewDataTable.addViewDataTable` and replaced it with `ViewDataTable.filterViewDataTable`. In general we always want to add components via the PHP files and to filter things we always use events. The `ViewDataTable.filterViewDataTable` is used and needed by Treemap plugin but will be also useful for other plugins